src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -76,7 +69,13 @@
 #include "plugins/adios/restart/LoadSpecies.hpp"
 #include "plugins/adios/restart/RestartFieldLoader.hpp"
 #include "plugins/adios/NDScalars.hpp"
-#include "plugins/common/stringHelpers.hpp"
+
+#include <pthread.h>
+#include <cassert>
+#include <sstream>
+#include <string>
+#include <list>
+#include <vector>
```

Comment:
QUESTION:
This re-placement of the includes was not part of the mentioned pull request / bug fixes. How did it end up here?
(Not an essential issue with this backport)